src/front/app/features/main/challenges/challenges.actions.spec.js

```diff
@@ -23,7 +22,7 @@ describe('action/challenges', () => {
         api = env.stub().returns(promise);
 
         sut = proxyquire('./challenges.actions', {
-            '../../common/api': api
+            '../../../common/api': api
```

Comment:
I think we should use absolute paths (resolve roots https://webpack.github.io/docs/configuration.html#resolve)